.gitignore

```diff
@@ -1,2 +1,3 @@
 composer.lock
 vendor
+.idea
```

Comment:
please do not clutter repositories with your ide-specific rules. Use a global gitignore file for that.